### DISCUSSION TITLE
question

### DISCUSSION THREAD

**Discussion by Mydak2281** [Charlese2/HostFixes#14] [Q&A] [answered]
When you put lever distance to 0 and die, will it change to normal distance, until you respawn???

**Comment by Charlese2** [comment, marked as answer]
If you set `Limit ship lever distance` to 1 or less, it disables the distance limit check in the mod. Dying won't change it to a normal distance.
https://github.com/Charlese2/HostFixes/blob/b1fa52d0083bbd304d231938c7fc4ab760c7d4b1/HostFixes/Plugin.cs#L699-L705
